.github/workflows/golangci-lint.yml

```diff
@@ -14,9 +14,9 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-go@v4
         with:
-          go-version: '^1.22'
+          go-version: '^1.23'
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v4
         with:
-          version: v1.57.1
+          version: v1.51.0
```

Comment:
Isn't this a downgrade?

Reply:
oops! yeah was meant to be the latest version, that's what I have on my machine. Thanks!